docs/spelling_wordlist

```diff
@@ -47,6 +50,7 @@ macOS
 Manjaro
 misconfigured
 Monetization
+multiline
```

Comment:
This should be either `multi-line`, or `MultilineTextInput` depending on context